init.lua

```diff
@@ -236,6 +238,11 @@ minetest.register_node("anvil:anvil", {
 			local meta = minetest.get_meta(pos)
 			local inv = meta:get_inventory()
 			inv:add_item("input", s)
+			local meta_description = s:get_meta():get_string("description")
+			if "" ~= meta_description then
+				meta:set_string("infotext", S("@1's anvil\n@2", owner, meta_description))
```

Comment:
`\n` cannot be translated. Please use `@n` or `@\n` instead.
https://github.com/minetest/minetest/blob/521a04222a71325ef217d1214febf5591919e169/doc/lua_api.txt#L3378-L3381

Reply:
thanks, will do